nav2_costmap_2d/plugins/static_layer.cpp

```diff
@@ -498,6 +508,17 @@ StaticLayer::dynamicParametersCallback(
         current_ = false;
       } else if (param_name == name_ + "." + "footprint_clearing_enabled") {
         footprint_clearing_enabled_ = parameter.as_bool();
+        if (!footprint_clearing_enabled_) {
```

Comment:
You can remove this log - I think the other one for restore cleared footprint is sufficient